test/e2e/metrics_adapter_test.go

```diff
@@ -319,7 +335,54 @@ func TestAggregatedMetricPermissions(t *testing.T) {
 	}
 }
 
+func TestMetricsServerRollout(t *testing.T) {
```

Comment:
Will this run on a cluster with prom-adapter and where we'll switch on mertricsserver? or will it run on a cluster that never knew about prom-adapter?

Reply:
We add a check to verify if MetricsServer featuregate enabled inside this func, it will proceed only if featuregate enabled so currently only on TechPreview cluster

```
if !f.IsFeatureGateEnabled(t, MetricsServerFeatureGate) {
		return
	}
```

Reply:
I updated using t.skip() instead of return

Reply:
Just the term `Rollout` + `...is deleted` make me think that the test would run after a migration from prom adapter to metricsserver.